src/components/molecules/LiveSchedule/LiveSchedule.tsx

```diff
@@ -18,23 +18,23 @@ const LiveSchedule: FC = () => {
     withEvents: true,
   });
 
-  const liveEvents = useMemo(() => {
+  const events = useMemo(() => {
     return relatedVenueEvents && relatedVenueEvents.length
       ? relatedVenueEvents.filter((event) => isEventLive(event))
       : [];
   }, [relatedVenueEvents]);
 
-  const hasLiveEvents = !!liveEvents.length;
+  const hasEvents = !!events.length;
```

Comment:
[musing] We use this 'convert to boolean' structure a lot, and I know I made at least 1 single character typo (`!` instead of `!!`) which we missed due to inferded typing.

A tiny generic helper function that takes `T` and returns `boolean` would be cool. eg.

```typescript
const toBoolean = <T>(item: T): boolean => !!item
```

Reply:
Great suggestion, here's a shot at it: https://github.com/sparkletown/sparkle/pull/783